flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/JoinITCase.scala

```diff
@@ -130,12 +136,14 @@ class JoinITCase(expectedJoinType: JoinType) extends BatchTestBase {
             case _ =>
           }
       }
-      Assert.assertTrue(haveTwoOp)
+      assertTrue(haveTwoOp)
```

Comment:
can we remove junit5 assertions and use only assertJ?

Reply:
Done. Currently, only assertJ is used.